### PR TITLE
Fix/175 frontend grid refactoring

### PR DESCRIPTION
**Pull request by mgaseta**
# Description

Fixed grid for all existing pages on SPAR.

Added some negative margins, which can be considered a bad practice, but since it is used only to bring elements closer together, I figured it is okay. Please shout out if you disagree and we can discuss other implementations. This practice is being used by Carbon too.

Fixes [#175](https://github.com/bcgov/nr-spar/issues/175)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manually tested and ran unit tests.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-197-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-197-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-197-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/src/views/Seedlot/CreateAClass/styles.scss [hunk 1]
```diff
@@ -1,9 +1,7 @@
-.create-a-class-seedlot-breadcrumb {
-  padding-left: 3.5rem;
-}
+@use '../../../theme/variables.scss' as vars;
```
Comment (RMCampos) [inline frontend/src/views/Seedlot/CreateAClass/styles.scss:1]: Do we need this import here?

**Review by RMCampos — CHANGES_REQUESTED** [review]
Great job! 💯 😄 only a question.

File: frontend/src/views/Seedlot/CreateAClass/styles.scss (hunk 1 above)
Comment (mgaseta) [reply frontend/src/views/Seedlot/CreateAClass/styles.scss]: No, we don't, forgot to remove it. Fixed [here](https://github.com/bcgov/nr-spar/pull/197/commits/ff7f2a5f53a98c95a5ba7868bdbe6aa0afa7f7a2)

**Review by RMCampos — APPROVED** [review]
LGTM. Great job!

**Pull request merged**